router/http.go

```diff
@@ -249,7 +249,7 @@ func (h *httpSyncHandler) Set(data *router.Route) error {
 	} else {
 		bf = backendFunc(r.Service, service.sc.Instances)
 	}
-	r.rp = proxy.NewReverseProxy(bf, h.l.cookieKey, r.Sticky, service, logger.New("service", r.Service))
+	r.rp = proxy.NewReverseProxy(bf, h.l.cookieKey, r.Sticky, r.DisableKeepAlives, service, logger.New("service", r.Service))
```

Comment:
This should probably be an options struct given the number of arguments building up.

Reply:
See 4299f290